flask_app/forms/upload_step3_form.py

```diff
@@ -20,7 +20,7 @@ class UploadStep3Form(FlaskForm):
 
     timepoint_count = IntegerField('timepoint_count', validators=[DataRequired()])
 
-    technique_type = SelectField('technique_type', choices=[
+    technique_types = SelectMultipleField('technique_types', choices=[
```

Comment:
Currently, we only allow one technique, which was a mistake. The backing code can handle multiple types, we just need to adjust the frontend.